src/udiskslinuxdriveata.c

```diff
@@ -470,6 +470,36 @@ static gboolean get_pm_state (UDisksLinuxDevice *device, GError **error, guchar
   return rc;
 }
 
+/**
+ * udisks_linux_drive_ata_is_active:
+ * @device: The ata device to get pm state from
+ * @error: Return location for error or %NULL
+ * @state: hex value of pm state
+ *
+ * Get the current power mode status from a drive.
+ *
+ * The format of @state is the result obtained from sending the
+ * ATA command CHECK POWER MODE to the drive.
+ * Known values include:
+ *     0x00 Standby
+ *     0x80 Idle
+ *     0xff Active/Idle
+ * Typically user interfaces will report "Drive is spun down" if @state is
+ * 0x00 and "Drive is spun up" otherwise.
+ *
+ * Returns: %TRUE if the operation succeeded, %FALSE otherwise.
+ */
+gboolean udisks_linux_drive_ata_is_active (UDisksLinuxDevice *device, GError **error, guchar state)
```

Comment:
`state` is unused here - it doesn't pass any value out since you don't pass a pointer in. You may well declare `state` as a local variable.